ts/state/selectors/messages.ts

```diff
@@ -192,10 +192,8 @@ export function useMessageSelected(messageId?: string) {
 
 export function useMessageSentWithProFeatures(messageId?: string) {
   const msgProps = useMessagePropsByMessageId(messageId);
-  const mockedFeatureFlags = useFeatureFlag('mockMessageProFeatures');
-  const proFeatures = mockedFeatureFlags.length
-    ? mockedFeatureFlags
-    : msgProps?.propsForMessage.proFeatures;
+  const mockedFeatureFlags = useDataFeatureFlag('mockMessageProFeatures');
+  const proFeatures = mockedFeatureFlags ?? msgProps?.propsForMessage.proFeatures;
```

Comment:
does `mockedFeatureFlags` get set to null when it is made empty after being set? If not you might want to add a `?.length` here